server/src/main/java/org/apache/druid/rpc/indexing/OverlordClientImpl.java

```diff
@@ -323,9 +323,6 @@ private <T> JsonParserIterator<T> asJsonParserIterator(final InputStream in, fin
     return new JsonParserIterator<>(
         jsonMapper.getTypeFactory().constructType(clazz),
         Futures.immediateFuture(in),
-        "", // We don't know URL at this point, but it's OK to use empty; it's used for logs/errors
```

Comment:
Is this the only usage of the new constructor? If yes, then we can probably just stick to what we had before.